src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -57,14 +58,14 @@ export interface ContentMessageProps extends Omit<MessageActions, 'onClickResetS
   hideHeader: boolean;
   hasMarker?: boolean;
   isFocused: boolean;
-  isLastDeliveredMessage: boolean;
   message: ContentMessage;
   onClickButton: (message: CompositeMessage, buttonId: string) => void;
   onRetry: (message: ContentMessage) => void;
   quotedMessage?: ContentMessage;
   selfId: QualifiedId;
   isMsgElementsFocusable: boolean;
   onClickReaction: (emoji: string) => void;
+  rightMarginWidth: number;
```

Comment:
In this implementation, `rightMarginWidth` gets calculated in `MessageList` and must be used to correctly position `MessageActionsMenu` (in old way, aligned to the screen, ignoring the new margin). It can be made as a context to not do this long chain-propsing